SimpleNotePad.java

```diff
@@ -0,0 +1,269 @@
+import java.awt.Dimension;

```

Comment:
A header would be a good addition to give an overview of the project, author, date, etc.